### DISCUSSION TITLE
Alt s does not work

### DISCUSSION THREAD

**Discussion by bq-wrongway** [helix-editor/helix#5925] [Q&A] [answered]
I tryed to use ALT s to split selection on new line, but it does not work, neither in gnome terminal or in foot terminal (my default). simply nothing happens, going from space ? and using command works.
I dont understand why, i am on the popOS.

in helix tutorial, 
i select several lines, and use alt s 
expected result 
selection gets split on new line
observed result
nothing happens.

Thanks!

**Comment by kirawi** [comment]
Next time please use the bug template.

**Comment by askreet** [comment]
It's almost certainly being intercepted by your terminal.

**Comment by bq-wrongway** [comment, marked as answer]
It appears that pop os default bindings were messing with the editor, even though in pop os, alt s did not exist meta s did , and untill i unbound this it would simply not work. i resolved my issue now.

**Comment by heliostatic** [comment]
I'm having the same issue on macOS + alacritty (and terminal, kitty, and iTerm). As far as I can tell, I don't have any system or application shortcuts overriding `alt-s`. Screen capture shown in iTerm.


https://user-images.githubusercontent.com/70898/230654678-46bc2975-0711-46ef-beb1-3939c3ff7f93.mp4

**Reply by heliostatic** [reply]
May be solved by https://github.com/helix-editor/helix/issues/2280 / this may be the OS level issue.

**Reply by heliostatic** [reply]
Yep, just a deeply annoying OS feature that has to be fixed per terminal emulator.
